src/Mod/Path/libarea/CMakeLists.txt

```diff
@@ -166,7 +166,7 @@ SET_PYTHON_PREFIX_SUFFIX(area)
 
 # this figures out where to install the Python modules
 execute_process(
-    COMMAND python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())"
+    COMMAND python -c "from sysconfig import get_path; print(get_path(\"purelib\"))"
```

Comment:
For me `from distutils.sysconfig import get_python_lib; print(get_python_lib())` gives the correct directory /usr/lib/python3/dist-packages while `from sysconfig import get_path; print(get_path("purelib"))` returns the non-existing directory /usr/lib/python3.6/site-packages

Reply:
May be related to this: https://wiki.debian.org/Python#Deviations_from_upstream
In Arch Linux I get the path to site-packages for both cases. I will investigate to get the correct path on Debian systems.

Reply:
>  I will investigate to get the correct path on Debian systems.  

@marioalexis84 any progress ?

Reply:
when you normally build from source I think it is a good idea to put things into site-packages, only debian packagers should be putting stuff into dist-packages (see the above link where they mention why they use dist-packages)

Reply:
on debian sid with python 3.11
```python
>>> from sysconfig import get_path; print(get_path("purelib"))
/usr/local/lib/python3.11/dist-packages
```

Reply:
ubuntu 22.04 with python 3.10:
```python
>>> from sysconfig import get_path; print(get_path("purelib"))
/usr/local/lib/python3.10/dist-packages
```

Reply:
and 
```python
>>> from sysconfig import get_path; print(get_path("purelib","deb_system"))
/usr/lib/python3/dist-packages
```

but I think this should only be used by packagers

Reply:
on 20.04 (which I think we still plan on supporting):
```python
>>> from sysconfig import get_path; print(get_path("purelib","deb_system"))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.8/sysconfig.py", line 525, in get_path
    return get_paths(scheme, vars, expand)[name]
  File "/usr/lib/python3.8/sysconfig.py", line 515, in get_paths
    return _expand_vars(scheme, vars)
  File "/usr/lib/python3.8/sysconfig.py", line 179, in _expand_vars
    for key, value in _INSTALL_SCHEMES[scheme].items():
KeyError: 'deb_system'
>>> from sysconfig import get_path; print(get_path("purelib"))
/usr/lib/python3.8/site-packages
```